test/sparsedir/sparse.jl

```diff
@@ -303,7 +310,8 @@ mfe22 = eye(Float64, 2)
 @test_throws ArgumentError sparsevec([3,5,7],[0.1,0.0,3.2],4)
 
 # issue #5169
-@test nnz(sparse([1,1],[1,2],[0.0,-0.0])) == 0
+# @test nnz(sparse([1,1],[1,2],[0.0,-0.0])) == 0
+# commented following change to sparse() in #14798, also see #12605, #9928, #9906, #6769
```

Comment:
> I can try some benchmarks with my Finite Element code.

I would be delighted to see those, particularly if your code can benefit from using `sparse!` in place of `sparse`. I should have a new version up shortly --- in the process of building and testing. Thanks and best!